domdiv/main.py

```diff
@@ -560,6 +583,57 @@ def read_card_data(options):
         Card.sets[s]['no_randomizer'] = Card.sets[s].get('no_randomizer', False)
         Card.sets[s]['fan'] = Card.sets[s].get('fan', False)
 
+    # Remove the Trash card. Do early before propagating to various sets.
+    if options.no_trash:
+        i = find_index_in_list(cards, 'card_tag', 'Trash')
+        if i is not None:
+            del cards[i]
+
+    # Repackage Curse cards into 10 per divider. Do early before propagating to various sets.
+    if options.curse10:
+        i = find_index_in_list(cards, 'card_tag', 'Curse')
+        if i is not None:
+            new_cards = []
+            cards_remaining = cards[i].getCardCount()
+            while cards_remaining > 10:
+                # make a new copy of the card and set count to 10
+                new_card = copy.deepcopy(cards[i])
+                new_card.setCardCount(10)
+                new_cards.append(new_card)
+                cards_remaining -= 10
+
+            # Adjust original Curse card to the remaining cards (should be 10)
+            cards[i].setCardCount(cards_remaining)
+            # Add the new dividers
+            cards.extend(new_cards)
+
+    # Create Start Deck dividers. 4 sets. Adjust totals for other cards, too.
+    # The card database contains one prototype divider that needs to be either duplicated or deleted.
+    # Do early before propagating to various sets.
+    i = find_index_in_list(cards, 'card_tag', 'Start Deck')
+    if i is not None:
+        new_cards = []
+        if options.start_decks:
```

Comment:
For clarity, can you move this `if` to the outer level and change the logic about deleting the start deck entry below to us a flag?

Maybe also print a warning if the option is specified but for some reason the `Start Deck` db entry isn't there.